ui/src/dropdown.test.tsx

```diff
@@ -128,6 +128,14 @@ describe('Dropdown.tsx', () => {
       expect(wave.args[name]).toMatchObject(['A', 'B', 'D'])
     })
 
+    it('Do not select all options on Select all - dropdown disabled', () => {
+      const { getByText } = render(<XDropdown model={{ ...defaultProps, values: ['A'], disabled: true }} />)
+
+      fireEvent.click(getByText('Select All'))
+
+      expect(wave.args[name]).toMatchObject(['A'])
```

Comment:
Let's make the initial state clear.
```suggestion
      expect(wave.args[name]).toMatchObject(['A'])

      fireEvent.click(getByText('Select All'))

      expect(wave.args[name]).toMatchObject(['A'])
```